packages/shared/src/lib/auth.ts

```diff
@@ -374,6 +374,41 @@ export const errorsToJson = <T extends string>(
 export const getErrorMessage = (errors: Message[]): string =>
   errors?.[0]?.text || '';
 
+export const getSettingsFlow = async (
+  id: string,
+): Promise<InitializationData> => {
+  const res = await fetch(`${authUrl}/self-service/settings/flows?id=${id}`, {
+    credentials: 'include',
+    headers: { Accept: 'application/json' },
+  });
+
+  return res.json();
+};
```

Comment:
This is basically the same as `getRegistrationFlow` note to clean this up.
(No need to change now as agreed, but we musn't forget.

Reply:
Ah, yes. I think the only difference is the endpoint, from `/registration` to `settings`. We can probably create an enum for available `AuthFlow`. We can probably start the refactoring on the ticket we have for kratos lib we have planned.

Reply:
Good idea let's add a task for it.

Reply:
I think this ticket we created before should be suitable for the things to work on. I'll add a comment referencing to our conversation here.

https://dailydotdev.atlassian.net/jira/software/projects/WT/boards/11?selectedIssue=WT-344